<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>977</width>
    <height>558</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QCalendarWidget" name="calendarWidget">
   <property name="geometry">
    <rect>
     <x>472</x>
     <y>191</y>
     <width>392</width>
     <height>236</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>416</width>
     <height>309</height>
    </size>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>110</y>
     <width>514</width>
     <height>58</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_7">
    <item>
     <spacer name="horizontalSpacer_6">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="maximumSize">
       <size>
        <width>10000000</width>
        <height>10000000</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(85, 85, 255);
font: italic 28pt &quot;Monotype Corsiva&quot;;

</string>
      </property>
      <property name="text">
       <string>Добавление олимпиады</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_7">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>39</x>
     <y>192</y>
     <width>425</width>
     <height>167</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="minimumSize">
         <size>
          <width>110</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>114</width>
          <height>27</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(85, 85, 255);
font: italic 15pt &quot;Monotype Corsiva&quot;;

</string>
        </property>
        <property name="text">
         <string>Название</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="maximumSize">
         <size>
          <width>114</width>
          <height>27</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(85, 85, 255);
font: italic 15pt &quot;Monotype Corsiva&quot;;

</string>
        </property>
        <property name="text">
         <string>Предмет</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_5">
        <property name="maximumSize">
         <size>
          <width>114</width>
          <height>27</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(85, 85, 255);
font: italic 15pt &quot;Monotype Corsiva&quot;;
</string>
        </property>
        <property name="text">
         <string>Место</string>
        </property>
        <property name="iconSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_2">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>27</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(85, 85, 255);
font: italic 15pt &quot;Monotype Corsiva&quot;;
</string>
        </property>
        <property name="text">
         <string>Время</string>
        </property>
        <property name="iconSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox">
        <property name="maximumSize">
         <size>
          <width>400</width>
          <height>27</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(85, 85, 255);
font: italic 15pt &quot;Monotype Corsiva&quot;;
</string>
        </property>
        <property name="text">
         <string>Дата</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLineEdit" name="lineEdit">
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>27</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>296</width>
            <height>27</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox">
          <property name="minimumSize">
           <size>
            <width>291</width>
            <height>27</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>291</width>
            <height>27</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_2">
          <property name="minimumSize">
           <size>
            <width>291</width>
            <height>27</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>291</width>
            <height>27</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTimeEdit" name="timeEdit">
          <property name="minimumSize">
           <size>
            <width>291</width>
            <height>27</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>291</width>
            <height>27</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>18</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
